jsk_fetch_robot/jsk_fetch_startup/config/install_supervisor.sh

```diff
@@ -15,4 +15,12 @@ for file in $(ls ./*.conf); do
     echo "copied $file to /etc/supervisor/conf.d"
 done
 
+# Enable jsk_dstat job to save the csv log under /var/log
+ln -s /home/fetch/Documents/jsk_dstat.csv /var/log/ros/jsk-dstat.csv
+
```

Comment:
what is jsk_dstat ?

Reply:
As I remember, @708yamaguchi had wanted to record cpu usage for navigation debugging. So he had added supervisor job to record performance with [dstat](https://github.com/dstat-real/dstat)

Reply:
where is jsk_dstat PR? please remove dstat related commit from this PR

Reply:
and create PR just for dstat for fetch robot

Reply:
I have created. https://github.com/jsk-ros-pkg/jsk_robot/pull/1825

Reply:
please remove dstat/supervisor commits from this PR

Reply:
Done.